### PR TITLE
Add customIcon property to Alert component

### PR DESCRIPTION
**Pull request by guykoh**

### CODE REVIEW

**Comment by guykoh** [comment]
@serdec, @vineethasok, @fneves will this create a new version of click-ui automatically? Or do I need to do anything else?

File: src/components/Icon/types.ts [hunk 1]
```diff
@@ -5,127 +5,129 @@ import { LogoName } from "../Logos/types";
 import { PaymentName, PaymentProps } from "../icons/Payments";
 
 export type IconSize = "xs" | "sm" | "md" | "lg" | "xl" | "xxl";
-export type IconName =
```
Comment (gjones) [inline src/components/Icon/types.ts:8]: Just out of interest, why did you do this?

**Review by gjones — APPROVED** [review]
LGTM

File: src/components/Icon/types.ts (hunk 1 above)
Comment (guykoh) [reply src/components/Icon/types.ts:8]: So I could use that list in storybook

File: src/components/Icon/types.ts [hunk 2]
```diff
@@ -5,127 +5,129 @@ import { LogoName } from "../Logos/types";
...
-  | "table"
-  | "taxi"
-  | "trash"
-  | "upload"
-  | "url"
-  | "user"
-  | "users"
-  | "warning"
-  | "waves";
+export const ICON_NAMES = [
```
[the remark this answers is not in the document]
Comment (guykoh) [reply src/components/Icon/types.ts:8]: How can you use the type as a list in storybook?

**Pull request merged**